integration/kind-test/scripts/run-helper.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Helper to run test and passing different Service names
+
+source ./integration/kind-test/scripts/common.sh
+
+# ClusterIP service test
+$KUBECTL_BIN apply -f "$CONFIGS/e2e-service.yaml"
+$KUBECTL_BIN apply -f "$CONFIGS/e2e-export.yaml"
+./integration/kind-test/scripts/run-tests.sh "$CLUSTERIP_SERVICE" "ClusterSetIP"
+
+# Remove ClusterIP service and imported service in order to prepare for next test
+IMPORTED_SVC=$($KUBECTL_BIN get svc -n $NAMESPACE | grep 'imported' | awk '{print $1}')
```

Comment:
Do we need to manually delete the imported service, and wouldn't it just get recreated? Shouldn't we just delete the export?

Reply:
Confirmed deleting export CRD works, will modify.